src/main/java/com/mcmoddev/mmdbot/core/TaskScheduler.java

```diff
@@ -63,5 +65,15 @@ public static void init() {
         TIMER.scheduleAtFixedRate(new MinecraftUpdateNotifier(), 0, fifteenMinutes);
         TIMER.scheduleAtFixedRate(new FabricApiUpdateNotifier(), 0, fifteenMinutes);
         TIMER.scheduleAtFixedRate(new ChannelMessageChecker(), 0, 1000 * 60 * 60 * 24);
+        TIMER.scheduleAtFixedRate(new TimerTask() {
+            @Override
+            public void run() {
+                if (ScamDetector.setupScamLinks()) {
+                    MMDBot.LOGGER.info("Successfully refreshed scam links");
+                } else {
+                    MMDBot.LOGGER.warn("Scam links could not be automatically refreshed");
+                }
+            }
+        }, 0, 1000 * 60 * 60 * 24 * 7);
```

Comment:
Could we have this run daily instead of weekly?

Reply:
https://github.com/MinecraftModDevelopment/MMDBot/pull/117#discussion_r787274384
See ^